CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## [0.1.11] - 2025-02-27
```

Comment:
If you don’t mind, could you remove the changes related to upgrading to 0.1.11? I’ll merge all the pending PRs and handle the upgrade myself afterward!